stdlib/universe/universe.flux

```diff
@@ -238,7 +245,7 @@ aggregateWindow = (
 // a threshold or are reset. In the case of a wrap/reset,
 // we can assume that the absolute delta between two points will be at least their non-negative difference.
 increase = (tables=<-, columns=["_value"]) => tables
-    |> difference(nonNegative: true, columns: columns)
+    |> difference(nonNegative: true, columns: columns, keepFirst: true, initialZero: true)
```

Comment:
My understanding was that `increase` doesn't default `keepFirst` to true? I'm ok if we expose it as an option, but I don't know if this should be the default.

Reply:
That is correct `keepFirst` was not in the default option to increase function. But to achieve the first row with all zero and satisfy all below four conditions in the `difference` function I have to introduce it to the increase function option. 

```
(keepFirst: false, initialZero: false) -> [10, 20, null, 10]
(keepFirst: true, initialZero: false) -> [null, 10, 20, null, 10]
(keepFirst: false, initialZero: true) -> [10, 20, 30, 10]
(keepFirst: true, initialZero: true) -> [0, 10, 20, 30, 10]
```